src/Aloha/Twilio/Twilio.php

```diff
@@ -13,13 +13,24 @@ public function __construct($config) {
     public function message($to, $message, $from=null) {
         $twilio = $this->getTwilio();
         // Send SMS via Twilio SDK
-        return $twilio->account->sms_messages->create(
+        return $twilio->account->messages->sendMessage(
             is_null($from) ? $this->config['from'] : $from,
             $to,
             $message
         );
     }
 
+    public function messageMMS($to, $message, $media, $from=null) {
```

Comment:
do you mind renaming it? I think the other name is better. And allows more easy upgrade from 1.x to 2.x later on.